lua/dap/ui/variables.lua

```diff
@@ -217,7 +217,7 @@ function M.resolve_expression()
 end
 
 
-function M.hover(resolve_expression_fn)
+local function do_hover_available_checks()
```

Comment:
```suggestion
local function check_hover_availability()
```

or

```suggestion
local function is_stopped_at_frame()
```

or

```suggestion
local function is_stopped_with_frame()
```


Not sure what's the best way to name this, but most methods do something, so the `do_` prefix feels a bit redundant and vague.